README.md

```diff
@@ -35,30 +40,36 @@ Environment variables
 
 The following environment variables are exposed for InfraMock:
 
-`EXPERIMENT_ID` : set the experiment ID, or leave unset for default experiment.
+`BIOMAGE_DATA_PATH`: where to get the experiment data to populate inframock's S3 and DynamoDB. If
+this is not set, it will default to `./data`.
 
-`POPULATE_MOCK`: whether localstack should be filled with a mocked PBMC data set. This is
-set to `true` by default, which means that InfraMock will try to use a locally running version of
-the `api` to populate the localstack DynamoDB database with using its `/experiments/generate` endpoint.
-For this to work, `api` **must** be running with `CLUSTER_ENV` set to `development`, which is the default behavior.
-
-`MOCK_EXPERIMENT_DATA_PATH`: where to get the mocked data for upload to the local S3 from. If
-this is not set, it will default to the Github URL where the dataset used for the `worker` unit
-test is located.
+`POPULATE_MOCK`: whether localstack should be filled with the data sets found in `BIOMAGE_DATA_PATH`.
+For this to work, `CLUSTER_ENV` must be set to `development`, which is the default behavior.
```

Comment:
you might have missed to state here that `CLUSTER_ENV` must be set to `development` in the api? Or at least this is how it used to be

Reply:
I'm actually confused by this. It stated that it will use the api somehow, but the api cannot be run without running inframock first because it will fail because it can't contact redis (started by inframock) so I don't really know what this `CLUSTER_ENV` is used for. Maybe it's not used anymore?

Reply:
I don't think `CLUSTER_ENV` references are really needed. I understand that if I ran with the default `CLUSTER_ENV`, I am using inframock. Otherwise, I am not using it and I don't care about its flags
https://github.com/biomage-ltd/api/blob/master/README.md#2-start-the-api

Reply:
What I'm talking about is this? `InfraMock will try to use a locally running version of
the `api` to populate the localstack DynamoDB database with using its /experiments/generate endpoint.z`. Because DynamoDB is populated by inframock's `src/app.py`, not relying on the api in any way to the best of my understanding.

Reply:
Ah! Thanks! I understand now. Definitely my observations agree on the API not being used for this: I have never started the API before inframock, and I have always had an initial experiment loaded.